<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterWindow</class>
 <widget class="QMainWindow" name="ConverterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Converter/NZRI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="assets/resources.qrc">
      <normaloff>:/main/backButton.png</normaloff>:/main/backButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="bit12">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bit9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="amplitudeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>Amplitude</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 15pt &quot;Avenir&quot;;</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="bg1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;#EAF0F8&quot;</string>
    </property>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 26pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Base conversion &amp; NZRI code graph</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bg2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>801</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;#EAF0F8&quot;</string>
    </property>
    <widget class="QLabel" name="numberEntryLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>55</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Insert number here:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numEntry">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 15pt &quot;Avenir&quot;;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>number in octal notation (base 8)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="analyzeButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="assets/resources.qrc">
       <normaloff>:/main/analyzeButton.png</normaloff>:/main/analyzeButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>160</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="baseConversionLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>125</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 italic 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Base conversion:</string>
     </property>
    </widget>
    <widget class="QLabel" name="binaryConversionLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Binary:</string>
     </property>
    </widget>
    <widget class="QLabel" name="hexConversionLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Hexadecimal:</string>
     </property>
    </widget>
    <widget class="QLabel" name="binaryConversionResultLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hexConversionResultLabel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>190</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 57 20pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0); font: 57 14pt &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>bg2</zorder>
   <zorder>bg1</zorder>
   <zorder>backButton</zorder>
   <zorder>bit12</zorder>
   <zorder>bit5</zorder>
   <zorder>bit8</zorder>
   <zorder>bit1</zorder>
   <zorder>bit11</zorder>
   <zorder>bit3</zorder>
   <zorder>bit2</zorder>
   <zorder>bit10</zorder>
   <zorder>bit6</zorder>
   <zorder>bit7</zorder>
   <zorder>bit4</zorder>
   <zorder>bit9</zorder>
   <zorder>amplitudeLabel</zorder>
   <zorder>timeLabel</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
